packages/beeq/src/components/page-title/scss/bq-page-title.variables.scss

```diff
@@ -14,8 +12,6 @@
    * @prop --bq-page-title--font-weight-title - Page title font weight for title
    * @prop --bq-page-title--font-weight-subtitle - Page title font weight for subtitle
    */
-  --bq-page-title--paddingX: theme(spacing.xxl4);
-  --bq-page-title--paddingY: theme(spacing.xl);
```

Comment:
The `paddingY` shouldn't be removed, only start/end padding. I will recommend using logical properties instead, meaning that you could set in the JSX, something like:

![CleanShot 2024-06-06 at 11 04 22@2x](https://github.com/Endava/BEEQ/assets/328492/ac48bc30-7cec-4bc5-84e3-6ac6263a819c)

Reply:
More on logical properties here:
https://www.sitepoint.com/css-logical-properties-guide/